modules/assim.batch/R/pda.utils.R

```diff
@@ -613,21 +617,55 @@ pda.generate.knots <- function(n.knot, n.param.all, prior.ind, prior.fn, pname)
 ##'
 ##' @return Nothing. Plot is generated and saved to PDF.
 ##'
-##' @author Ryan Kelly
+##' @author Ryan Kelly, Istem Fer
 ##' @export
 pda.plot.params <- function(settings, params.subset, prior.ind) {
   # *** TODO: Generalize for multiple PFTS
   pdf(file.path(settings$pfts$pft$outdir, 
         paste0('mcmc.diagnostics.pda', settings$assim.batch$ensemble.id, '.pdf')))
-    dm <- as.mcmc(params.subset)
+   params.mcmc.list <- as.mcmc.list(lapply(params.subset, mcmc))
 
-    plot(dm)
-    summary(dm)
+   plot(params.mcmc.list)
+   
+    dm <- do.call("rbind",params.subset)
```

Comment:
_If_ pecan is already using the `dplyr` package, consider using `bind_rows`, which is highly efficient and clearer.